src/logger/fileWriter.c

```diff
@@ -149,13 +152,13 @@ static int write_samples_data(const LoggerMessage *msg)
         size_t count = msg->sample->channel_count;
 
         if (NULL == sample) {
-                pr_warning("Logger: null sample record\r\n");
+                pr_warning("_RCP_BASE_FILE_: null sample record\r\n");
```

Comment:
Ahh... yes.  Its a macro.  Lets figure out exactly what we want and how we want to define it... then we can implement a special macro for it and have it just work.